hypha/apply/funds/templates/funds/includes/generic_primary_actions.html

```diff
@@ -0,0 +1,13 @@
+{% load primaryactions_tags %}
+
+{% if request.user|should_display_primary_actions_block:object %}
```

Comment:
This will be fun to maintain & debug. Cant think of something better at this point in time.

Reply:
Yeah, I know. Not pretty but I can't think of a better solution either